cli/command/formatter/disk_usage.go

```diff
@@ -26,7 +26,7 @@ const (
 	uniqueSizeHeader  = "UNIQUE SiZE"
 )
 
-// DiskUsageContext contains disk usage specific information required by the formater, encapsulate a Context struct.
+// DiskUsageContext contains disk usage specific information required by the formatter, encapsulate a Context struct.
```

Comment:
also found in https://github.com/docker/docker/blob/master/cli/command/formatter/image.go#L23,
could you update that as well?